portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,30 @@
 /**
  * Adds a random greeting to the page.
  */
```

Comment:
Update the JSDoc.

Can use a single line JSDoc if you prefer.

```javascript
/** Adds a random quote to the page. */
```

Reply:
Changes made